### PR TITLE
[WIP] Added php-task

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q             | A
| ------------- | ---
| License       | MIT

This PR add the php-task bundle to enable asynchronous task execution.

### CODE REVIEW

File: php-task/task-bundle/1.2/config/packages/task.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+task:
+    storage:                 doctrine # One of "array"; "doctrine"
+    run:
+        mode:                'off' # One of "off"; "listener"
+    locking:
+        enabled:             false
+        # ttl:                 600
```
Comment (Nyholm) [inline php-task/task-bundle/1.2/config/packages/task.yaml]: Try to avoid having configuration as comments like this.

**Comment by Nyholm** [comment]
You also need to update composer.json and mark you package as “symfony-bundle”. 
The static validation and Travis needs to be green so I can merge it.

**Comment by wachterjohannes** [comment]
@Nyholm I have seen that :) will update that in the evening!

**Comment by wachterjohannes** [comment]
@Nyholm this should now be valid

**Comment by stof** [comment]
@wachterjohannes you still have a bunch of commented config setting, which should not be part of the file added in the project (the console_path for instance is something that 99% of users will never need to change as your bundle should be able to have a default value suited for all projects following the Symfony structure).

The file added by the recipe becomes part of the userland project, and so is maintained by users after that. Using it as documentation is not a good idea (as documentation should be maintained by you and updated when the bundle changes)

**Comment by wachterjohannes** [comment]
@stof ou shit :) completely forgotten that! thanks for the hint

**Comment by wachterjohannes** [comment]
@Nyholm and the bundle will also be added to the kernel?

**Comment by Nyholm** [comment]
Yes, it will =)

**Comment by wachterjohannes** [comment]
@Nyholm ok then its not necessary :) i would like to see some default config in the application when installing a bundle - but thats ok if not!

**Pull request closed without merging**